dali/test/python/test_dali_variable_batch_size.py

```diff
@@ -1081,6 +1083,36 @@ def test_subscript_dim_check():
     check_pipeline(data, single_op_pipeline, operator_fn=fn.subscript_dim_check, num_subscripts=1)
 
 
+def test_crop_argument_from_external_source():
```

Comment:
I think we can remove the previous variable batch size test for this operator, because apparently it failed to detect a rather obvious issue.